ols/app/endpoints/ols.py

```diff
@@ -40,7 +40,7 @@ def conversation_request(
     referenced_documents: list[str] = []
 
     # TODO: retrieve proper user ID from request
-    user_id = "user1"
+    user_id = constants.DEFAULT_USER_UID
```

Comment:
it is odd to call this a "default" user id.  There is no default user id for OLS.  this is a sample value, at best.  I would not expect any OLS code outside of test logic, to make use of this value.  After all, there is no such thing as a valid "default" user.

when we get into production with auth enabled and have requests coming in w/ no user id associated, we do not want to fallback to some default value(which will then fail, but in a less obvious way).  We want to fail because the user id is a required parameter.

So if the purpose of this logic is to allow OLS to not fail when we don't have user id information being passed in, we need to think a little more about how we want to handle that during dev/testing, without impacting how OLS behaves in production.

Reply:
yup I know, there's OLS-378 that are pretty easy to get done (talked about it with Nimer couple hours ago).
So it will go away (no default user at all).